vignettes/howtogenefilter.Rmd

```diff
@@ -0,0 +1,190 @@
+---
+title: "Using the genefilter function to filter genes from a microarray dataset"
+author:
+- name: Robert, Gentleman
+  affiliation: Vignette Author's Institution
+- name: Khadijah Amusat
+  affiliation: Vignette translation from Sweave to R Markdown / HTML
+date: March 09, 2023
```

Comment:
Please use the following inline r code to generate the date whenever the file is updated:

    `r format(Sys.Date(), "%B %d, %Y")`